src/v/ssx/future-util.h

```diff
@@ -352,6 +352,29 @@ repeat_until_gate_closed(seastar::gate& gate, Func&& func) noexcept {
     });
 }
 
+/// \brief Repeats the passed func in a detached fiber until the gate is closed
+/// or abort is triggered.
+///
+/// \param gate Gate object to use.
+/// \param as Abort source to use.
+/// \param func function to invoke.
+template<class Func>
+inline void repeat_until_gate_closed_or_aborted(
```

Comment:
`inline` is unnecessary here?

Reply:
:shrug: not sure. Each call generates a unique template instantiation, so not inlining it looks like a net overhead to me. I guess it will be inlined regardless the modifier. What are your thoughts on this?

Reply:
it wasn't really a question. i think it is unncessary. it is only necessary if the template is fully specialized.